chcemvediet/apps/accounts/migrations/0005_socialapp_remove_twitter_and_linkedin.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+def forward(apps, schema_editor):
+    SocialApp = apps.get_model(u'socialaccount', u'SocialApp')
+    SocialApp.objects.filter(pk__in=[3, 4]).delete()
```

Comment:
Nezmazalo to zaznamy z tabulky `socialaccount_socialaccount`. Zrejme tam nie je FK na `socialaccount_socialapp`. Zislo by sa zmazat aj tie.

Reply:
:ok: